### PR TITLE
feat(sqlite): add vela resource ddl

### PR DESCRIPTION
**Pull request by jbrockopp**
Part of the effort for https://github.com/go-vela/community/issues/248

Related to https://github.com/go-vela/server/pull/342

This adds the Sqlite [data definition language (DDL)](https://en.wikipedia.org/wiki/Data_definition_language) for Vela.

All of the DDL is contained in the `github.com/go-vela/server/database/sqlite/ddl` package.

The DDL was copied from [the existing `../../server/database/ddl/sqlite`](https://github.com/go-vela/server/tree/master/database/ddl/sqlite) old package structure.

This is an ongoing effort to make the `../../server/database` package conform to standards set in other packages.

### CODE REVIEW

File: database/sqlite/ddl/user.go [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+	token          TEXT,
+	hash           TEXT,
+	favorites      TEXT,
+	active         BOOLEAN,
+	admin          BOOLEAN,
+	UNIQUE(name)
+);
+`
+
+	// CreateRefreshIndex represents a query to create an
```
Comment (wass3r) [inline database/sqlite/ddl/user.go:26]:
```suggestion
	// CreateUserRefreshIndex represents a query to create an
```

File: database/sqlite/ddl/worker.go [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+	address         TEXT,
+	routes          TEXT,
+	active          TEXT,
+	last_checked_in	INTEGER,
+	build_limit     INTEGER,
+	UNIQUE(hostname)
+);
+`
+
+	// CreateWorkersHostnameAddressIndex represents a query to create an
```
Comment (wass3r) [inline database/sqlite/ddl/worker.go:25]:
```suggestion
	// CreateWorkerHostnameAddressIndex represents a query to create an
```

**Review by wass3r — COMMENTED** [review]
Couple of typos - otherwise 👍🏼

File: database/sqlite/ddl/user.go (hunk 1 above)
Comment (jbrockopp) [reply database/sqlite/ddl/user.go]: fixed 👍

File: database/sqlite/ddl/worker.go (hunk 2 above)
Comment (jbrockopp) [reply database/sqlite/ddl/worker.go]: fixed 👍

**Review by wass3r — APPROVED** [review]

**Review by KellyMerrick — APPROVED** [review]
lgtm

**Pull request merged**